chia/data_layer/data_layer_api.py

```diff
@@ -15,6 +16,7 @@ class DataLayerAPI:
 
     log: logging.Logger
     data_layer: DataLayer
+    metadata: ClassVar[ApiMetadata] = ApiMetadata()
```

Comment:
do we still need this even though there are no methods marked up by `metadata.request()`?

Reply:
To fulfill the typing protocol I would think.  The whole file needs to be deleted though but it's not quite trivial to make the API optional in `Service` IIRC.